script.R

```diff
@@ -6,11 +6,10 @@ remotes::install_github("insightsengineering/verdepcheck")
 remotes::install_github("r-lib/rcmdcheck#196") # TODO: remove when merged / linked issue fixed
 
 args <- commandArgs(trailingOnly = TRUE)
-path <- normalizePath(file.path(".", args[1]))
-build_args <- strsplit(args[2], " ")[[1]]
-if (is.na(build_args) || build_args == "") build_args <- character(0)
-check_args <- strsplit(args[3], " ")[[1]]
-if (is.na(check_args) || check_args == "") check_args <- character(0)
+path <- normalizePath(file.path(".", Sys.getenv("VERDEPCHECK_REPOSITORY_PATH")))
```

Comment:
Those params are expected to be empty for the simplest use cases and we should definitely handle such cases. It was enabled by the if statement you deleted. Can you try add them back again?

Reply:
Yes, the problem that I was trying to solve was that when these are passed as arguments to the script, and build/check args are empty, `strategy` argument is incorrectly interpreted as `build_args` argument. For example [here](https://github.com/insightsengineering/teal.code/actions/runs/4616351142/jobs/8161253602).
I thought that maybe environment variable would be a good way to solve it, since then the order of arguments doesn't matter and they can be empty.

Reply:
ok got it - we would have to name the parameters when calling the script. Let me do that

Reply:
As discussed, the method to execute the script in a way that passes empty strings as arguments works fine.
But there's [one failure for `max` strategy](https://github.com/insightsengineering/teal.code/actions/runs/4616926109/jobs/8162553197).

Reply:
[Workflow for teal.code](https://github.com/insightsengineering/teal.code/actions/runs/4617182445) is passing.